packages/api-db-mongodb/src/db/user.ts

```diff
@@ -70,6 +72,23 @@ export class MongoDBUserAdapter implements DBUserAdapter {
     }
   }
 
+  public async createUserFromTempUser(tempUser: TempUser): Promise<OptionalUser> {
+    const newUser = await this.createUser({
+      input: {
+        email: tempUser.email,
+        name: tempUser.name,
+        address: tempUser.address,
+        preferredName: tempUser.preferredName,
+        active: true,
+        roleIDs: [],
+        properties: [],
+        emailVerifiedAt: null
+      },
+      password: crypto.randomBytes(48).toString('hex')
```

Comment:
Why to hex? Shouldn't it be to .toString('base64') or .toString('utf8')? The Problem is if you use hex the complexity is reduced to 16 possibilities per char. This would not matter if we would use an modern hash function but since we use bcrypt and bcrypt cuts of after 72 bytes. Which means we loose some generated bytes ;-) Probably not security relevant.

Reply:
I copy pasted the snippet from Nicos code. But toString() is limited to base 16, which is hex ;)

Reply:
.toString('base64') is surly not limited to base 16 ;-) its base 64 ;-) And .toString('utf8') even base 256 ;-)

Reply:
changed it. it's buffer encoding which allows "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex"